src/main/java/seedu/address/model/tag/UniqueTagList.java

```diff
@@ -78,6 +80,42 @@ public boolean contains(Tag toCheck) {
     }
 
     /**
+     * If the tag name is valid and the tag present in the list.
+     * @param tagName
+     * @return the index of the tag in the list
+     * @throws IllegalValueException
+     */
+
+    public int contains(String tagName) throws IllegalValueException {
```

Comment:
This should be private and should be named `indexOf` (or similar) to reflect its actual function if it is still required.